config/hub/default/kustomization.yaml

```diff
@@ -6,7 +6,7 @@ namespace: ramen-system
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: ramen-
+namePrefix: ramen-hub-
```

Comment:
When we discussed, I believe we said only "hub".  But having seen a few places that use dr-hub, I like it equally, especially, it is consistent to say ramen-dr-hub-operator vs ramen-dr-cluster-operator.

Reply:
I changed `drhub` in the leader election to `hub`, in an upcoming commit on this PR. To keep it consistent.

Name change across to call it drhub we can discuss further.